src/legend/svgLegend.ts

```diff
@@ -844,10 +858,11 @@ export class SVGLegend implements ILegend {
         arrows
             .attr("transform", (d: NavigationArrow) => translate(d.x, d.y))
             .select("path")
-            .attr({
-                "d": (d: NavigationArrow) => d.path,
-                "transform": (d: NavigationArrow) => d.rotateTransform
-            });
+            .attr(
+                "d", (d: NavigationArrow) => d.path)
```

Comment:
Since it's less than 120 symbol we should write this function call into a single line